### DISCUSSION TITLE
try to install the library

### DISCUSSION THREAD

**Discussion by rraulf** [mkdocstrings/mkdocstrings#682] [Q&A] [answered]
I try to install the library (as in #680 @matanox) and get an error:

ERROR: Could not find a version that satisfies the requirement mkdocstring (from versions: none)
ERROR: No matching distribution found for mkdocstring

What is the solution?

Thanks!

**Comment by pawamoy** [comment, marked as answer]
> mkdocstring

You're missing the last "s": `mkdocstrings` :slightly_smiling_face: 

Please avoid opening *both* a discussion and [issue](https://github.com/mkdocstrings/mkdocstrings/issues/683) :upside_down_face:
